code/datums/components/storage/concrete/bag_of_holding.dm

```diff
@@ -9,7 +9,7 @@
 		if(safety != "Proceed" || QDELETED(A) || QDELETED(W) || QDELETED(user) || !user.canUseTopic(A, BE_CLOSE, iscarbon(user)))
 			return
 		var/turf/loccheck = get_turf(A)
-		to_chat(user, "<span class='danger'>The Bluespace interfaces of the two devices catastrophically malfunction!</span>")
+		to_chat(user, "<span class='danger'>The bluespace interfaces of the two devices catastrophically malfunction!</span>")
```

Comment:
Surely Bluespace is supposed to be capitalized as it is essentially a name of a place?

Reply:
... no?

Its usage is more like that of the words "quantum" or "magic", as far as I'm aware.

Reply:
I might be wrong but in this case "Bluespace" would a substitute for a "pocket dimension" and would therefore be capitalized.

Reply:
After looking through the code a bit it seems that it is indeed "bluespace" and not "Bluespace". My mistake.